src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -55,11 +55,11 @@ const ExplorationViewPage = () => {
     },
     Heatmap: {
       toolbarControls: [
-        <HeatmapSettings />,
+        <HeatmapSettings componentType={COMPONENT_TYPE} />,
         <RemoveButton />,
       ],
       component: (width, height) => (
-        <HeatmapPlot experimentId={experimentId} width={width} height={height} />
+        <HeatmapPlot componentType={COMPONENT_TYPE} experimentId={experimentId} width={width} height={height} />
```

Comment:
why are you passing `COMPONENT_TYPE` in here? It is not used, `HeatmapPlot` uses `COMPONENT_TYPE` directly